firewall/firewall_interfaces.go

```diff
@@ -4,11 +4,20 @@ package firewall
 type QueryHandlerInterface interface {
 	CheckQuery(sqlQuery string) error
 
+	Refresh()
+
 	AddQueries(queries []string)
 	RemoveQueries(queries []string)
 
 	AddTables(tables []string)
 	RemoveTables(tables []string)
+
+	AddRules(rules []string)
+	RemoveRules(rules []string)
+
+	GetActiveQueries() []string
```

Comment:
Our handler interface grow and grow and in a future will be very big monolitic interface. But at start it was designed like simple abstract interface for different handlers with simple function `CheckQuery` and encapsulated logic in custom handler itself. 
Anyway what mean `Active` in name? Now it's just added queries/tables/rules to handler. So will be enough to name `GetQueries`. Because `active` tell us that handler has some different queries too

Reply:
Ok. Renamed